flow/matrix_decomposition_unittests.cc

```diff
@@ -2,11 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#include "flutter/fml/build_config.h"
-
-#if defined(OS_WIN)
```

Comment:
What is this change for?

Reply:
This symbol is only required on Windows (for `M_PI` etc.) and this is the only place we actually wrap it in a platform check, but also the place we benefit least from doing so (unittest). We could wrap it everywhere, but figured it would be nice to keep the code consistent so people don't end up doing a mix of the two approaches.